proto/teleport/lib/teleterm/v1/service.proto

```diff
@@ -185,6 +185,13 @@ service TerminalService {
 
   // ConnectToDesktop is a bidirectional stream for the desktop connection.
   rpc ConnectToDesktop(stream ConnectToDesktopRequest) returns (stream ConnectToDesktopResponse);
+  // AttachDirectoryToDesktopSession opens a directory for a desktop session and enables file system operations for it.
+  // If there is no active desktop session associated with the specified desktop_uri and login,
+  // the RPC returns an error.
+  //
+  // This RPC does not automatically share the directory with the server (it does not send a SharedDirectoryAnnounce message).
+  // It only registers file system handlers for processing file system-related TDP events.
+  rpc AttachDirectoryToDesktopSession(AttachDirectoryToDesktopSessionRequest) returns (AttachDirectoryToDesktopSessionResponse);
```

Comment:
This makes the RPC sound way more fancy than what it actually does. When I first encountered it, I assumed it'd be doing some kind of a desktop access magic to "attach" the dir.

I think we can be much more explicit here. The purpose of this RPC is to enable a stream started through ConnectToDesktop to handle filesystem TDP messages. It provides the path of the shared directory that those messages are going to operate on.

"Attach" is not necessarily a bad name, but in my opinion it implies that there's more stuff happening compared to what actually happens. Maybe `SetSharedDirectoryForDesktopSession` would be good enough?

Reply:
Yeah, I felt that `AttachDirectoryToDesktopSession` sounds like some serious stuff is happening under the hood, but I couldn't find a better name. `SetSharedDirectoryForDesktopSession` sound good 👍